index.html

```diff
@@ -244,12 +243,9 @@ <h2>Success Criteria</h2>
       <a href="#advancement">§3.1</a> to demonstrate adequate
       <a href="https://www.w3.org/Consortium/Process/#implementation-experience">implementation experience</a>.</p>
 
-    <p>Each Recommendation-track Technical Report:</p>
+    <p><span class="todo">extend all itemized lines?</span>Each Recommendation-track Technical Report:</p>
 
     <ul>
-      <li>SHOULD contain a section detailing all known security and privacy implications for implementers, Web authors, and end users.</li>
-      <li>SHOULD have an associated testing plan, starting from the earliest drafts.</li>
-      <li>SHOULD contain a section on accessibility that describes the benefits and impacts, including ways that features defined in the Technical Report can be used to address them, and recommendations for maximising accessibility in implementations.</li>
```

Comment:
Clearer as a list in my opinion.

Reply:
charter template has de-itemized list of these, and I'd follow that.

Reply:
Why? Doesn't that make it harder to read and understand? Sounds like a fix to push back into the charter template.

Reply:
yeah, I find this a bit clearer and more explicit. Also, all of these items are also mentioned below on lines 240-244

Reply:
Please feel free to file an issue to [the template](https://github.com/w3c/charter-drafts/blob/gh-pages/charter-template.html), of course.
(I personally prefer itemized list for this kind of conditions...)

Reply:
changed to keep itemized, where the same meaning line is stated in new template.